src/toisto/ui/cli.py

```diff
@@ -97,7 +97,17 @@ def add_practice_command(self) -> None:
             "practice a language, for example type `%(prog)s practice --target fi --source en` to "
             "practice Finnish from English"
         )
-        self.add_command("practice", "Practice a language.", command_help)
+        parser = self.add_command("practice", "Practice a language.", command_help)
+        if (default := self.config.get("practice", "progress_update")) == "0":
+            default += " (never)"
+        parser.add_argument(
+            "-p",
+            "--progress-update",
+            metavar="{frequency}",
+            type=int,
+            default=0,
+            help=f"show a progress update while practicing after each {{frequency}} quizzes; default: {default}",
```

Comment:
"while practicing" is superfluous since this is an option for the practice command.